habitat/tasks/rearrange/multi_task/composite_sensors.py

```diff
@@ -117,62 +155,25 @@ def update_metric(self, *args, episode, task, observations, **kwargs):
 
 
 @registry.register_measure
-class CompositeReward(Measure):
-    """
-    The reward based on where the agent currently is in the hand defined solution list.
-    """
-
-    cls_uuid: str = "composite_reward"
+class CompositeSparseReward(Measure):
+    cls_uuid: str = "composite_sparse_reward"
 
     @staticmethod
     def _get_uuid(*args, **kwargs):
-        return CompositeReward.cls_uuid
-
-    def __init__(self, sim, config, *args, **kwargs):
-        super().__init__(**kwargs)
-        self._sim = sim
-        self._config = config
-        self._prev_node_idx = None
-
-    def reset_metric(self, *args, episode, task, observations, **kwargs):
-        task.measurements.check_measure_dependencies(
-            self.uuid,
-            [CompositeNodeIdx.cls_uuid],
-        )
+        return CompositeSparseReward.cls_uuid
 
+    def reset_metric(self, *args, **kwargs):
         self.update_metric(
             *args,
-            episode=episode,
-            task=task,
-            observations=observations,
             **kwargs,
         )
 
     def update_metric(self, *args, episode, task, observations, **kwargs):
-        self._metric = 0.0
-        node_measure = task.measurements.measures[CompositeNodeIdx.cls_uuid]
-
-        node_idx = node_measure.get_metric()["node_idx"]
-        if self._prev_node_idx is None:
-            self._prev_node_idx = node_idx
-        elif node_idx > self._prev_node_idx:
-            self._metric += self._config.STAGE_COMPLETE_REWARD
-
-        if task.forced_node_task is None:
-            cur_task_cfg = task.get_inferrred_node_task()._config
-        else:
-            cur_task_cfg = task.forced_node_task._config
+        self._metric = self._config.SLACK_REWARD
```

Comment:
It looks like it only adds a slack reward, not sure what this reward is or why it is sparse (slack rewards tend to be dense)